extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,134 +25,119 @@ limitations under the License.
 
-[tip type="important"]
-**Important** [`<amp-analytics>`](../amp-analytics/amp-analytics.md) is [currently unsupported](https://github.com/ampproject/amphtml/issues/15807) on pages users land on through `<amp-next-page>`.
-Tracking page views is supported through [`<amp-pixel>`](../../builtins/amp-pixel.md) or `<amp-analytics>` on the host page.
-[/tip]
+### Footer Recommendation Box
 
-### Analytics triggers
+If the user reaches the end of a page before the next has loaded, or if the
+next page fails to load or if the maximum number of recommendations is reached (as specified by `max-pages`) a box will be displayed with links to the remaining pages. See the Styling section for information on how to customize the default recommendation box.
 
-Partial support for analytics is included through the initial host page via two separate events. These are triggered by `<amp-next-page>` and you can track them in your [amp-analytics](https://amp.dev/documentation/components/amp-analytics) config:
-
-| Event                  | Fired when                                               |
-| ---------------------- | -------------------------------------------------------- |
-| `amp-next-page-scroll` | The user scrolls to a new page                           |
-| `amp-next-page-click`  | The user click on an article from the recommendation box |
-
-Both of the `triggers` provide the variables `fromUrl` and `toUrl` referring to the previous and current pages. They can be used as follows:
+The footer / recommendation box can also be templated via `amp-mustache` or other templating engines, and will be passed an array of the remaining `pages`, each being a page object containing a `title`, `url` and `image`. Example:
 
 ```html
-<amp-analytics>
-  <script type="application/json">
-    {
-      "requests": {
-        "nextpage": "https://foo.com/pixel?RANDOM&toURL=${toURL}"
-      },
-      "triggers": {
-        "trackScrollThrough": {
-          "on": "amp-next-page-scroll",
-          "request": "nextpage"
-        },
-        "trackClickThrough": {
-          "on": "amp-next-page-click",
-          "request": "nextpage"
-        }
-      }
-    }
-  </script>
-</amp-analytics>
+<amp-next-page src="https://example.com/config.json">
+  <div footer class="my-custom-footer">
+    Here are a few more articles:
+    <template type="amp-mustache">
+      <div class="footer-content">
+        {{#pages}}
+        <span class="title">{{title}}</span>
+        <span class="url">{{url}}</span>
+        <span class="image">{{image}}</span>
+        {{/pages}}
+      </div>
+    </template>
+  </div>
+</amp-next-page>
 ```
```

Comment:
Big question about this section:
What if I do not specify this in `<amp-next-page`? Is this auto done for me? Is this only done so I can style it/customize it?  Is this different than a regular footer? Can I specify both? Whats going on with this `footer` attribute in the `<div>` element? 

Why is `footer / recommendation` box slashed like that? Do I specify something different for the footer than the "regular" recommendation box? Is there a difference?

Reply:
Hey, I hope the offline discussion clarified some of these questions. For posterity, if you do not specify this then yes, it's done for you in a very similar way (although without the `Here are a few more articles` text and with slightly different styles). You can't specify both since you only get the default footer if `<amp-next-page>` does not find an element with the `footer` attribute inside your component. And yes, this is done so that you can style it and maybe add more text below the title of the article, change the layout, etc which can't be done by simply styling the default one in CSS.